### DISCUSSION TITLE
Removal of highly correlated features

### DISCUSSION THREAD

**Discussion by mglowacki100** [autogluon/autogluon#3074] [Ideas]
I was looking at the documentation, but I didn't find information that highly corellated features are removed during pre-processing (only  constant features are removed).
I wonder if it is possible to add this step to autogluon, it could be activated by setting parameter like `correlation_threshold` to e.g. `0.9`. I've noticed that in `eda` module this functionality is already implemented, so it shouldn't be a lot of work to extend autogluon predictor. Additional benefit is that feature importance makes more sense when there is no highly corellated features.
What do you think about it?

**Comment by Innixma** [comment]
Moving to issue for further discussion
